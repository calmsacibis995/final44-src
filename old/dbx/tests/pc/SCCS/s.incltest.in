h19024
s 00011/00000/00000
d D 5.1 90/06/25 15:37:01 bostic 1 0
c date and time created 90/06/25 15:37:01 by bostic
e
u
U
t
T
I 1
whereis v
whereis p
whatis p
whereis T
whatis includes.T
whereis OT
whatis included.OT
whereis Blah
stop in Blah
run
where
E 1
