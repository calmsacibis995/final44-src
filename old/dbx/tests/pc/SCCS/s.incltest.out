h40634
s 00019/00000/00000
d D 5.1 90/06/25 15:37:03 bostic 1 0
c date and time created 90/06/25 15:37:03 by bostic
e
u
U
t
T
I 1
reading symbolic information ...
includes.v
includes.p
var p : includes.T;
included.T includes.T
type T = record
    a : integer;
    b : integer;
end;
included.OT includes.OT
type OT = integer;
included.Blah
[1] stop in Blah
[1] stopped in Blah at line 6 in file "included.p"
    6   begin
Blah(), line 6 in "included.p"
program(), line 13 in "includes.p"
main(0x1, 0x7fffe7e4, 0x7fffe7ec) at 0x6d

E 1
